### PR TITLE
feat(linter): eslint-plugin-jsx-a11y autocomplete-valid

### PR DESCRIPTION
**Pull request by yossydev**
partof: https://github.com/oxc-project/oxc/issues/1141

- doc: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/autocomplete-valid.md
- code: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/rules/autocomplete-valid.js
- test: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/__tests__/src/rules/autocomplete-valid-test.js

### CODE REVIEW

**Comment by yossydev** [comment]
@camc314 
> Hey @yossydev !
> 
> this is looking good - need any help here to get this over the line?

Sorry, I failed to open it!🙏
Please give us a review!💪

File: crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    /// <input autocomplete="invalid-value" />
+    ///
+    /// // Good
+    /// <input autocomplete="name" />
+    /// ```
+    AutocompleteValid,
+    correctness
+);
+
+static VALID_AUTOCOMPLETE_VALUES: Lazy<HashSet<&'static str>> = Lazy::new(|| {
```
Comment (camc314) [inline crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs:47]: can we change this to a phf_set please

**Review by camc314 — COMMENTED** [review]
Looks great!

once we've updated it to use phf this is good to go!

File: crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs (hunk 1 above)
Comment (yossydev) [reply crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs]: Thank you!
https://github.com/oxc-project/oxc/pull/1901/commits/bcb260f579ae154a64e68842af237ec0c7998ff1 ← How about this way?

**Review by camc314 — APPROVED** [review]
Thank you!

File: crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs (hunk 1 above)
Comment (camc314) [reply crates/oxc_linter/src/rules/jsx_a11y/autocomplete_valid.rs]: yep that's perfect!

**Pull request merged**